test_requirements.txt

```diff
@@ -1,4 +1,5 @@
 # requirements for CI
+wheel
 sympy == 1.7
```

Comment:
Should this be updated? Also I noticed that @eric-wieser 's PR failed against SymPy master which might indicate that GAlgebra (at least the tests) is only compatible with 1.7 but will have issues somewhere between 1.7 and master.

Reply:
When I removed the restriction to 1.7, I got [a lot of failures](https://github.com/moble/galgebra/actions/runs/5868440428/job/15911280790) that I'm guessing are related to sympy just organizing negative signs differently than it used to.  This means that explicit comparisons to old results (as are done with nbval and in test_test.py) can fail spuriously.  I suppose the thing to do would be to check more carefully than I have that most of those errors are indeed just sign-organization errors, re-run the notebooks, and update test_test.py.  But I'm gonna suggest leaving that for a later PR.